ledger/internal/eval_blackbox_test.go

```diff
@@ -533,6 +533,8 @@ func endBlock(t testing.TB, ledger *ledger.Ledger, eval *internal.BlockEvaluator
 	require.NoError(t, err)
 	err = ledger.AddValidatedBlock(*validatedBlock, agreement.Certificate{})
 	require.NoError(t, err)
+	rndBQ := ledger.Latest()
```

Comment:
I would have guessed we need to wait for `Latest()+1`, as I thought Latest() was the already committed block.  It's easy to get confused here, I might be confused!

If you're also unsure, perhaps just try waiting for `Latest()+1`. If it hangs forever, I guess we know `Latest()` was correct.

Reply:
ah that makes sense, let me check on my local end: oops it is hanging forever.

I was trying to ensure that everything in `ledger.bq` is written to DB, I will add a note there to explain things. The `Latest()` calls `blockQueue`'s `latest`, and it gives the committed round + rounds in queue.

Reply:
Ah, ok, `Latest()` makes sense then. Thanks.